src/pycram/pose_generator_and_validator.py

```diff
@@ -17,6 +17,13 @@
 
 class PoseGenerator:
     current_orientation_generator = None
+    """
+    If no orientation generator is given, this generator is used to generate the orientation of the robot.
+    """
+    override_orientation_generator = None
```

Comment:
why are these 2 different fields? Just default it to the ordinary orientation generate and if the user wants a different one he can set that, or am I missing something?